bin/admin/install

```diff
@@ -1127,6 +1127,47 @@ if [ "$nothing" = 0 ]; then
         action_done
     fi
 
+    # The Bastion > v3.04.00
+    action_doing "Migrating account creation metadata to new format where applicable"
+    at_least_one_changed=0
+    at_least_one_error=0
+    for accounthome in $(getent passwd | grep ":$basedir/bin/shell/osh.pl$" | cut -d: -f6); do
+        # if the old AND the new files exists, don't do anything: we won't delete the old file
+        # because we want our users to be able to roll back to an old version, and as this file doesn't
+        # hurt anything (it'll just get ignored by new versions), no point in inserting an incompatible change here.
+        if [ -f "$accounthome/accountCreate.comment" ] && ! [ -f "$accounthome/config.creation_info" ]; then
+            at_least_one_changed=$((at_least_one_changed + 1))
+            # build a config.creation_info file from the data stored in the legacy accountCreate.comment
+            perl -MJSON -MPOSIX -e '
+                my %h;
+                open(my $oldfh, "<", $ARGV[0]) or die $!;
+                while (<$oldfh>) {
+                    $h{$1}=$2 if m{^([^=]+)=(.+)};
+                }
+                close($oldfh);
+                open(my $newfh, ">", $ARGV[1]) or die $!;
+                print $newfh encode_json({
+                    by => $h{CREATED_BY},
+                    bastion_version => $h{BASTION_VERSION},
+                    time_utc => POSIX::strftime("%a %Y-%m-%d %H:%M:%S UTC", gmtime($h{CREATION_TIMESTAMP})),
+                    time_local => POSIX::strftime("%a %Y-%m-%d %H:%M:%S %Z", localtime($h{CREATION_TIMESTAMP})),
```

Comment:
Why both utc & local ?

Reply:
Can't you store ISO 8601 DateTime ?

Reply:
The main / important way to store it is via the `timestamp` field, and this is what is used in the code when it's required to take decisions, make calculations, etc. The `time_utc` and `time_local` fields are just human-friendly strings that are here for the admin that would directly read the file on disk (this is bad, but if some do it, let's be nice to them), and it's also outputted as-is in `accountInfo` and in the resulting JSON.